src/main/java/com/google/sps/servlets/startGameServlet.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.servlets;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+ 
+/** Create game entity and connect it to the user, then update the user entity to include the game id
+    and store both in datastore when the user creates a new game.
+ */
+@WebServlet("/start-game")
+public class startGameServlet extends HttpServlet {
+ 
+  DatastoreService datastore;
+ 
+  public startGameServlet(){
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+ 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String gameName = request.getParameter("game-name");
```

Comment:
Should probably check for existence of a valid game name here. What if someone sends over a game name that already exists (which might not be an issue globally, but would be an issue for a specific user... I probably dont want to have two active games with the same name)? or an empty game name? we should be able to detect these issues and then act accordingly. 

Additionally, this is a great candidate for test code. This could be its own method and we can have some nice test cases written around it.

Reply:
Okay, so that function is written, and I have a call to it commented out in this file, because it won't work until we incorporate user entities.